mmdeploy/codebase/mmdet3d/deploy/mmdetection3d.py

```diff
@@ -1,116 +1,359 @@
 # Copyright (c) OpenMMLab. All rights reserved.
-from typing import Optional, Union
+from copy import deepcopy
```

Comment:
put these codes in `voxel_detection.py` would be better, we might add other task in mmdet3d.

Reply:
done.